### ISSUE TITLE
Update overlay background color according to Photon

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#5148]
The overlay background is blue but it should be gray according to the Photon spec: https://design.firefox.com/photon/patterns/overlay.html.

![](https://design.firefox.com/photon/images/overlay/overlay-style.svg)

**Comment by AlexandraMoga** [comment]
I have looked at the Write a review component as it seems to be the only one that is using an overlay currently. 

![image](https://user-images.githubusercontent.com/31961530/40792631-143a9718-6503-11e8-81e3-4408a81fc999.png)
![image](https://user-images.githubusercontent.com/31961530/40792650-1dc7d944-6503-11e8-9427-227a4053b40b.png)


Verifed as fixed on AMO-dev with FF60, WIn10x64 and Android 7.0
